test/services/leaderboard-stat-getter.test.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable import/first */
+jest.mock('forest-express');
+jest.mock('../../src/utils/orm');
+import Sequelize from 'sequelize';
+import ForestExpress from 'forest-express';
+import Orm from '../../src/utils/orm';
+
+const sequelize = new Sequelize({ dialect: 'postgres' });
```

Comment:
[SCHAFOLDING] Having these test preparations under functions with clear names would help to identify what your are doing such as:
`generateModels()..
generateSchemas()...
`